packages/salesforcedx-utils-vscode/src/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const SETTING_CLEAR_OUTPUT_TAB = 'clearOutputTab';
+export const SFDX_CORE_CONFIGURATION_NAME = 'salesforcedx-vscode-core';
```

Comment:
What's your thought here having these two identical strings with different names?

Reply:
I decided that since they represent 2 different things, that we should have 2 constants, even though they are the same string.  

I wanted anyone changing these in the future to be more aware of what this constant represents.